src/main/host/thread_ptrace.c

```diff
@@ -371,6 +372,10 @@ static pid_t _threadptrace_fork_exec(const char* file, char* const argv[], char*
             if (prctl(PR_SET_TSC, PR_TSC_SIGSEGV, 0, 0, 0) < 0) {
                 error("prctl: %s", g_strerror(errno));
             }
+            // Set the working directory
+            if (workingDir != NULL && chdir(workingDir) < 0) {
```

Comment:
Probably doesn't make sense to silently ignore a NULL workingDir, since that'd be a programming error. Maybe `utility_assert(workingDir)`?

Reply:
Done